ReferralPlease/ReferralPlease/ViewController.swift

```diff
@@ -154,7 +154,7 @@ extension ViewController: WKNavigationDelegate {
     
     func goToFirstTimeLoginView(_ user: User) {
         let storyboard = UIStoryboard (name: "Main", bundle: nil)
-        guard let firstTimeViewController = storyboard.instantiateViewController(withIdentifier: "firstTimeViewController") as? FirstTimeLoginViewController else
+        guard let firstTimeViewController = storyboard.instantiateViewController(withIdentifier: "FirstTimeLoginFormViewController") as? FirstTimeLoginViewController else
```

Comment:
@chloevo26 this line will raise an exception. Can you fix this? And don't we still need to go to first time view controller to ask if the user is a mentor or mentee?

Reply:
I fix this problem